client/my-sites/sharing/connections/service.jsx

```diff
@@ -163,36 +166,36 @@ module.exports = React.createClass( {
 		if ( ! connection ) {
 			// When triggering a refresh from the primary action button, find
 			// the first broken connection owned by the current user.
-			connection = serviceConnections.getRefreshableConnections( this.props.service.name )[0];
+			connection = serviceConnections.getRefreshableConnections( this.props.service.ID )[0];
```

Comment:
Since we're already touching this line, let's update the `[ 0 ]` to get rid of an eslint warning.